### PR TITLE
Enable MSAA on Wave GL context

### PR DESCRIPTION
**Pull request by MortimerGoro**

### CODE REVIEW

File: app/src/wavevr/cpp/native-lib.cpp [hunk 1]
```diff
@@ -52,7 +52,7 @@ int main(int argc, char *argv[]) {
 
   // Must initialize render runtime before all OpenGL code.
   WVR_RenderInitParams_t param;
-  param = { WVR_GraphicsApiType_OpenGL, WVR_RenderConfig_Timewarp_Asynchronous };
+  param = { WVR_GraphicsApiType_OpenGL, WVR_RenderConfig_Timewarp_Asynchronous | WVR_RenderConfig_MSAA};
```
Comment (bluemarvin) [inline app/src/wavevr/cpp/native-lib.cpp]: I'm confused, what is getting anti-aliased? Our FBOs already have 4x multi sampling. Does this cause the distortion pass to also multi-sample? I tried looking at the docs but it wasn't very clear to me.

**Comment by bluemarvin** [comment]
I'm still messaging with HTC over what this option actually does.

**Comment by bluemarvin** [comment]
I never got a response on this on the Wave forum.

**Comment by MortimerGoro** [comment]
Ok, closing this. According to this thread it doesn't have any effect on Vive Focus: https://community.viveport.com/t5/Vive-Wave-SDK/WVR-RenderConfig-MSAA/td-p/26307

And we are achieving MSAA with the custom FBOs.

**Comment by bluemarvin** [comment]
> Ok, closing this. According to this thread it doesn't have any effect on Vive Focus: https://community.viveport.com/t5/Vive-Wave-SDK/WVR-RenderConfig-MSAA/td-p/26307

Weird, I never got an email that they finally responded.

**Pull request closed without merging**